### ISSUE TITLE
'Colors' and 'custom' pop up given for color legends of graph is not accessible via keyboard in windows.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#114144]
### What happened?

**User Impact:**
Keyboard navigation users will be impacted because the ‘Colors’ and ‘Custom’ pop-ups in the color legend cannot be opened or navigated using the keyboard, blocking customization and data interpretation.

**Actual Result:**
'Colors' and 'custom' pop up given for color legends of graph is not accessible via keyboard in windows.

**Refer Attachment:**

1. 'Colors' and 'custom' pop up given for color legends of graph is not accessible via keyboard in windows.png
2. 'Colors' and 'custom' pop up given for color legends of graph is not accessible via keyboard in windows.mp4

https://github.com/user-attachments/assets/61bc4e06-ddb3-475d-b669-7318d2807fc2
<img width="974" height="561" alt="Image" src="https://github.com/user-attachments/assets/62b2a11b-756c-472d-a094-6c759fc709a4" />

### What did you expect to happen?

'Colors' and 'custom' pop up given for color legends of graph should be accessible via keyboard in windows.

### Did this work before?

No

### How do we reproduce it?

1. Open the URL: https://play.grafana.org/d/ktMs4D6Mk/bar-charts-and-pie-charts?orgId=1&from=now-6h&to=now&timezone=utc in edge browser in windows.
2. Bar Charts and pie charts page will appear.
3. Press tab key to reach to the 'Popular JS Framework' graph.
4. Press tab key to reach to the color legends.
5. Verify whether 'Colors' and 'custom' pop up given for color legends of graph is accessible via keyboard or not.

### Environment (with versions)?

Grafana: Grafana Cloud (fast)
OS: windows 11 Version 25H2 (OS Build 26200.7262)
Browser: Edge (Version 143.0.3650.28)
URL: https://play.grafana.org/d/ktMs4D6Mk/bar-charts-and-pie-charts?orgId=1&from=now-6h&to=now&timezone=utc

### Grafana platform?

I use Grafana Cloud

**Comment by eledobleefe** [comment]
Given that the `ColorPicker` works as expected when using the keyboard, and that the legend lives under `packages/grafana-ui/src/components/VizLegend` that is owned by DataViz, we understand this issue is for @grafana/dataviz-squad. Let us know if this is not right.

**Comment by msftedad** [comment]
"I'm not sure what the claim is, can you elborate? There is nothing that we bundle here ourselves. The environment is Grafana, and the last time that I tested (in November) this happens in Grafana Playground as well. If the issue is with a sub-component of Grafana - this should be fixed. How is being a part of @grafana/dataviz-squad and VizLegend, not make it a Grafana issue?

If I'm misunderstanding and the question is if an issue should be opened for @grafana/dataviz-squad, then I would follow what you saying as you are the experts and know best.

**Comment by ashharrison90** [comment]
fixed by https://github.com/grafana/grafana/pull/117038
